config/routes.rb

```diff
@@ -6,6 +6,7 @@
   get 'foods/index'
   get 'inventories/index'
   get 'recipes/index'
+  resources :recipes, only: [:index, :show, :destroy]
```

Comment:
We can now remove `get 'recipes/index'`.

Reply:
Hey @piero-vic 
Removed the unused routes and updated the navbar that used the link